.github/workflows/qiita-ci.yml

```diff
@@ -7,11 +7,12 @@ on:
 jobs:
   # derived from https://github.com/actions/example-services/blob/master/.github/workflows/postgres-service.yml
   main:
+    # 7/16/24: confirm current ubuntu-latest is still 22.04.
```

Comment:
needed?

Reply:
It's informative, but I can take it out.